app/models/namespace_group_link.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# entity class for NamespaceGroupLink
+class NamespaceGroupLink < ApplicationRecord
+  has_logidze
+  acts_as_paranoid
+
+  before_validation :set_namespace_type
+
+  belongs_to :group, class_name: 'Group'
+  belongs_to :namespace, class_name: 'Namespace'
+
+  validates :group_id, uniqueness: { scope: [:namespace_id] }
+
+  validates :group_access_level, inclusion: { in: Member::AccessLevel.all_values_with_owner },
+                                 presence: true
+
+  validates :namespace_type,
+            inclusion: {
+              in: %w[Group Project]
```

Comment:
**nitpick (blocking):** This is a little picky, but can this be changed to `[Group.sti_name, Namespaces::ProjectNamespace.sti_name]` that way if for some reason they are updated it stays in sync.

Reply:
Sure thing! Updated in 4300e64efad5ad78957caac152655279f12b672a